src/react/components/icons/Swipe.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const Swipe = () => (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
+        <line x1="15" y1="24" x2="16.5" y2="24" fill="none" stroke="#000" strokeLinejoin="bevel" strokeWidth="1.75"/>
```

Comment:
We'll need to have the colors defined with css/styled-components intead (`stroke="#000"` etc).